lib/react_client/react_interop.dart

```diff
@@ -5,11 +5,13 @@
 library react_client.react_interop;
 
 import 'dart:html';
+import 'dart:js_util';
 
 import 'package:meta/meta.dart';
 import 'package:js/js.dart';
 import 'package:react/react.dart';
-import 'package:react/react_client.dart' show ComponentFactory;
+import 'package:react/react_client.dart' show ComponentFactory, ReactJsComponentFactoryProxy;
+import 'package:react/react_client.dart';
```

Comment:
Duplicate import of the previous line; instead, the `show` should be updated with what we need from this lib.